usecases/auth/authorization/types.go

```diff
@@ -30,6 +31,9 @@ const (
 	UPDATE = "U"
 	// DELETE Represents the action to delete a resource.
 	DELETE = "D"
+
+	ROLE_SCOPE_ALL   = "(C)|(R)|(U)|(D)"
+	ROLE_SCOPE_MATCH = "MATCH"
```

Comment:
should we just make them all explicit to avoid confusion? Eg "ALL" instead of "(C)|(R)|(U)|(D)"? Or is there anything we would gain by using "(C)|(R)|(U)|(D)"

Reply:
the idea that all cover all operations individually